mycroft/configuration/mycroft.conf

```diff
@@ -192,6 +192,12 @@
     "stand_up_word": "wake up"
   },
 
+  // Settings used by microphone to set recording timeout
+  "microphone": {
+    "recording_timeout": 10.0,
+    "recording_timeout_with_silence": 3.0
```

Comment:
Let's move this to the listener config so it's grouped with the other microphone configs

Reply:
moved in newer commit